planning/behavior_path_planner/autoware_behavior_path_start_planner_module/src/clothoid_pull_out.cpp

```diff
@@ -1445,8 +1445,9 @@ std::optional<PullOutPath> ClothoidPullOut::plan(
       std::make_pair(initial_velocity, acceleration));
     pull_out_path.partial_paths.push_back(clothoid_path);  // Use validated and cropped path
 
-    pull_out_path.start_pose =
-      clothoid_path.points.empty() ? start_pose : clothoid_path.points.front().point.pose;
+    pull_out_path.start_pose = resampled_combined_path.points.empty()
+                                 ? start_pose
+                                 : resampled_combined_path.points.front().point.pose;
```

Comment:
memo: 

`resampled_combined_path.points.front()` is start of clothoid_path segment.